### PR TITLE
Change some attributes to only_local.

### PR DESCRIPTION
**Pull request by surechen**
Modified according to https://github.com/rust-lang/compiler-team/issues/505.

r? @lcnr

### CODE REVIEW

File: compiler/rustc_feature/src/builtin_attrs.rs [hunk 1]
```diff
@@ -720,7 +722,8 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
     ),
     // Ensure the argument to this function is &&str during const-check.
     rustc_attr!(
-        rustc_const_panic_str, Normal, template!(Word), WarnFollowing, INTERNAL_UNSTABLE
+        rustc_const_panic_str, Normal, template!(Word),
+        WarnFollowing, INTERNAL_UNSTABLE
```
Comment (lcnr) [inline compiler/rustc_feature/src/builtin_attrs.rs:726]: spurious formatting changes, please revert and rebase

**Review by lcnr — COMMENTED** [review]
r=me after handling the nit

File: compiler/rustc_feature/src/builtin_attrs.rs (hunk 1 above)
Comment (surechen) [reply compiler/rustc_feature/src/builtin_attrs.rs]: > spurious formatting changes, please revert and rebase

Done. Thank you.

**Pull request merged**